python/ray/serve/tests/test_standalone.py

```diff
@@ -100,26 +105,29 @@ def test_detached_deployment(ray_cluster):
     ray.init(head_node.address)
     assert ray.get_runtime_context().job_id != first_job_id
 
-    serve.create_backend("g", lambda _: "world")
-    serve.create_endpoint("g", backend="g")
-    assert ray.get(serve.get_handle("g").remote()) == "world"
+    @serve.deployment
+    def g(*args):
+        return "world"
+
+    g.deploy()
+    assert ray.get(g.get_handle().remote()) == "world"
 
 
 @pytest.mark.skipif(sys.platform == "win32", reason="Failing on Windows")
 @pytest.mark.parametrize("detached", [True, False])
 def test_connect(detached, ray_shutdown):
-    # Check that you can call serve.connect() from within a backend for both
+    # Check that you can make API calls from within a deployment for both
     # detached and non-detached instances.
     ray.init(num_cpus=16)
     serve.start(detached=detached)
 
-    def connect_in_backend(_):
-        serve.create_backend("backend-ception", connect_in_backend)
+    @serve.deployment
+    def connect_in_backend(*args):
+        connect_in_backend.options(name="backend-ception").deploy()
```

Comment:
Test failure here: `ValueError: route_prefix /connect_in_backend is already registered.`

Reply:
👍